README.md

```diff
@@ -583,6 +583,7 @@ Table of Contents
   * [glitch.com](https://glitch.com/) — Free unlimited public/private hosting with features such as code sharing and real-time collaboration
   * [heroku.com](https://www.heroku.com/) — Host your apps in the cloud, free for single process apps
   * [KintoHub](https://www.kintohub.com/) — KintoHub is an all-in-one platform to combine and deploy your backend services, websites, cron jobs, databases and everything your app needs in one place.
+  * [Krucible](https://usekrucible.com) — Krucible is a platform for creating Kubernetes clusters for testing and development. Free tier accounts come with 25 clusters each month.
```

Comment:
"25 clusters" means what? how many credits/hours of run time? And I presume this is 25/month right?

Reply:
Yes, 25 cluster-hours per month. I'll update the PR.